vic/vic_run/src/lakes.eb.c

```diff
@@ -26,6 +26,7 @@
 
 #include <vic_def.h>
 #include <vic_run.h>
+#include <vic_driver_shared.h>
```

Comment:
that's fine too.  As long as we don't have any driver code sitting in `vic_run`.